bundle/bind.go

```diff
@@ -70,12 +70,18 @@ func (e *executor) Bind(
 			}
 		}
 
-		creds, err := ExtractCredentials(
+		credBytes, err := runtime.Provider.ExtractCredentials(
```

Comment:
Just want to make sure I understand this correctly. The runtime is returning `[]byte` that represent the credentials returned from the bundle and the bundle package is responsible for turning that into the `ExtractedCredentials` type.